bin/ytool

```diff
@@ -10,6 +10,8 @@ parser.add_argument("--file", "-f", action="store",
     help="Name of the chart file to change", required=True)
 parser.add_argument("--set-string", "-s", action='append', nargs=2, default=list(),
     metavar=('path', 'value'), help='Set string value for path')
+parser.add_argument("--set-string-keyvalue", "-skv", action='append', nargs=5, default=list(),
```

Comment:
if possible use a single letter for the short switch, i.e., `-k` instead of  `-skv`